operator/src/main/resources/scripts/utils.sh

```diff
@@ -418,3 +418,14 @@ function getAdminServerUrl() {
   fi
   echo ${admin_protocol}://${AS_SERVICE_NAME}:${ADMIN_PORT}
 }
+
+#
+# adjustPath
+#   purpose: Prepend $PATH with $JAVA_HOME/bin if $JAVA_HOME is set
+#
+function adjustPath() {
+  if [ ! -z ${JAVA_HOME} ]; then
```

Comment:
Maybe add a condition here to see if $JAVA_HOME/bin has already been added to the PATH?
Something like
if ! [[ $PATH =~ "${JAVA_HOME}/bin" ]]
then
  \# change the PATH
fi